### PR TITLE
Move onboarding tags from public to internal API

### PR DESCRIPTION
**Pull request by citizen428**
## What type of PR is this? (check all applicable)

- [X] Refactor

## Description

Move from `/api/tags/onboarding` to `/tags/onboarding` as this was not supposed to be part of our public API.

## Related Tickets & Documents

#6255 

- [X] yes

## Added to documentation?

- [X] no documentation needed

### CODE REVIEW

**Comment by rhymes** [comment]
The problem with this, is that the API is already publicly accessible - see https://dev.to/api/tags/onboarding.

Yeah, we know it shouldn't be used and it probably isn't (how do we know?) but we shouldn't remove publicly available endpoints anyway. We can adjust that in a better designed API v1 ;-)

API v0 is full of weird oddities, I'd keep this in

**Comment by mstruve** [comment]
@rhymes makes a great point. What about keeping the API but also adding the internal endpoint and using that internal one for our front end code? That way if we need to change the endpoint for our internal calls we can without worry about the API.

**Comment by citizen428** [comment]
@rhymes I'm a bit confused, as this is one of the endpoints you listed in the #6255.

**Comment by rhymes** [comment]
@citizen428 totally my fault, sorry! I didn't notice it wasn't authenticated. I don't think we break anything by removing it but I also think we should live with our idiosincracies in this case :D Going to remove it from the list anyhow

File: app/controllers/internal/configs_controller.rb [hunk 1]
```diff
@@ -47,6 +47,6 @@ def clean_up_params
   end
 
   def bust_relevant_caches
-    CacheBuster.bust("/api/tags/onboarding") # Needs to change when suggested_tags is edited
+    CacheBuster.bust("/tags/onboarding") # Needs to changed when suggested_tags is edited
```
Comment (atsmith813) [inline app/controllers/internal/configs_controller.rb]: I think this comment was originally fine. Or if we wanted to change it, how about something like:
`# Needs to be changed when suggested_tags is edited`

Comment (citizen428) [reply app/controllers/internal/configs_controller.rb]: It was, somehow I read it as "need to be" anyway. 🤦‍♂

**Comment by citizen428** [comment]
@rhymes I'm not saying this because I made the PR, that was quick anyway. But I think we should just remove it. It wasn't supposed to be public, v0 is not a stable API, it'd be gone in v1 anyway and it wasn't even documented yet.

**Comment by citizen428** [comment]
@benhalpern Do you have any opinion on keeping/removing this?

**Pull request merged**